app/Http/Controllers/User/BookController.php

```diff
@@ -267,4 +271,21 @@ public function destroy($id)
     {
         //
     }
+
+    public function getDetailData(Request $request)
+    {
+        $req = [
+            'type' => $request->type,
+            'book_id' => $request->book_id,
+        ];
+        $data = $this->bookUser->getDetailData($req);
+        if (Auth::check()) {
+            $followingIds = Auth::user()->followings->pluck('id')->toArray();
+        } else {
+            $followingIds = NULL;
+        }
+        $type = $request->type;
+
+        return view('layout.section.user_list', compact('data', 'type', 'followingIds'));
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.PHP.LowerCaseConstant.Found</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>TRUE, FALSE and NULL must be lowercase; expected "null" but found "NULL"

Reply:
@framgiaciv3 (clap)

Reply:
@dangminhtruong anh check sao nó báo lỗi chỗ này đi anh

Reply:
@BunnyPi04  Xem trên trang CIV3 cho chính xác em ạ. Con này chưa ổn định, đợi có thời gian bọn anh sửa lại.... 
File này bị lỗi ở đây thì phải này https://civ3-dev.framgia.vn/repositories/PHPEducation/Fbook/builds/11/violation/phpcs/10

Reply:
dạ em vừa fix lại rồi ạ. mà k biết có được không :(

Reply:
@HaiHaChan Bé fix hết chỗ lỗi phpcs tròng này nè em https://civ3-dev.framgia.vn/repositories/PHPEducation/Fbook/builds/13/violation

Reply:
Add chatwork anh chụp hình gửi cho nhé...